cmake/AddFdbTest.cmake

```
@@ -184,15 +184,84 @@ function(create_test_package)
       file(COPY ${file} DESTINATION ${CMAKE_BINARY_DIR}/packages/${dest_dir})
     endforeach()
   endforeach()
-  set(tar_file ${CMAKE_BINARY_DIR}/packages/correctness.tar.gz)
+  set(tar_file ${CMAKE_BINARY_DIR}/packages/correctness-${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}.tar.gz)
```

Only build correctness package, if not a Valgrind build
The version number also should include the patch version X.Y.Z
```suggestion
  if (NOT USE_VALGRIND)
  set(tar_file ${CMAKE_BINARY_DIR}/packages/correctness-${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}.tar.gz)
```